runtime/memprof.c

```diff
@@ -2233,7 +2233,11 @@ CAMLprim value caml_memprof_start(value lv, value szv, value tracker)
     one_log1m_lambda = MIN_ONE_LOG1M_LAMBDA; /* negative infinity */
   }
 
-  one_log1m_lambda_v = caml_copy_double(one_log1m_lambda);
+  lambda_v = caml_alloc_shr(Double_wosize, Double_tag);
+  Store_double_val(lambda_v, lambda);
```

Comment:
Maybe this remark is silly, but still: what is the point of allocating `lambda_v` (be it in the major or in the minor heap) if we never use it after all? Before the patch, `lambda_v` did not exist.